### ISSUE TITLE
AND does not work as intended

### ISSUE THREAD

**Issue by Gloix** [JoeRoddy/firestation#6]
If i test the query:

    select * from users where email = 'john.doe@example.com'

I get 1 record (assuming that email exists) with the `email` field matching that of the query, and also with another field `country='US'`. If I extend the query like this:

    select * from users where email = 'john.doe@example.com' AND country = 'US'

I get zero results.

**Comment by JoeRoddy** [comment]
Surprised I didn't notice this.  Thanks @Gloix .

You can use a lower-case 'and' for now and it should work as intended.  I'll have this fixed for the next release.

**Comment by Gloix** [comment]
Wow I never tested with the lowercase version. Thank you, I'll use that for now.

**Comment by Gloix** [comment]
Speaking of which, I tested the OR operator as well, with no luck as well. Uppercase and lowercase 😕.

**Comment by JoeRoddy** [comment]
Right, maybe we should make another issue for that at some point.  I have a page with some of the supported/non-supported features [here](https://docs.firestation.io/#where-clauses-filtering-data).  It's possible it's out of date.

**Comment by Gloix** [comment]
Oh right, I'm sorry, I didn't see that. Maybe outputting an error in response to a query with missing operators will prevent people from reporting things like that :)
